Add accessibilityTraits to support screen readers

cooperka opened the pull request:
Using `accessibilityTraits="button"`, for example, allows screen readers like Voiceover or TalkBack to behave more appropriately. See the [React Native](https://facebook.github.io/react-native/docs/accessibility.html) or [Apple](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/iPhoneAccessibility/Making_Application_Accessible/Making_Application_Accessible.html#//apple_ref/doc/uid/TP40008785-CH102-SW7) guides for more info.

FaridSafi commented:
Thank you @cooperka! I like this new feature

I am just curious about 2 things :
- Why Bubble instead of MessageText and MessageImage?
- What about Send button?

cooperka commented:
Good catch! I tested everything except for actually typing a message 😎. I've updated Send button too.

I added to Bubble because currently that's the component that actually receives the touch event when you tap it (see screenshot).

![simulator screen shot aug 25 2016 17 53 18](https://cloud.githubusercontent.com/assets/2047062/17987227/d4634dc2-6aec-11e6-8827-c5bf13d7e993.png)

FaridSafi commented:
thank you!

cooperka commented:
Thank you! I appreciate your quick responses 🍻
